portieris.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: portieris
+  version: 0.13.19
+  epoch: 0
+  description: A Kubernetes Admission Controller for verifying image trust.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/IBM/portieris.git
+      tag: v${{package.version}}
+      expected-commit: d0705039d060ab11b8fa598b79647ac0008af57e
+
+  - runs: |
+      VERSION=v${{package.version}} make portieris
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/usr/bin/
+      install -m755 ./portieris ${{targets.destdir}}/usr/bin/
+
+data:
+  - name: components
+    items:
+      # Only the keys matter
+      portieris:
+
+update:
+  enabled: true
+  github:
+    identifier: IBM/portieris
+    strip-prefix: v
+    tag-filter: v0.13.
```

Comment:
tag-filter can be removed here given upstream is using semver naming and no version streams.